service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/StønadsstatistikkMapper.kt

```diff
@@ -24,71 +23,85 @@ internal class StønadsstatistikkMapper(private val clock: Clock) {
             stonadstype = Statistikk.Stønad.Stønadstype.SU_UFØR,
             sakId = vedtak.behandling.sakId,
             aktorId = aktørId.toString().toLong(),
-            sakstype = vedtakstype(vedtak.vedtakType),
+            sakstype = vedtakstype(vedtak),
             vedtaksdato = vedtak.opprettet.toLocalDate(zoneIdOslo),
-            vedtakstype = vedtakstype(vedtak.vedtakType),
-            vedtaksresultat = when (vedtak.vedtakType) {
-                VedtakType.SØKNAD -> Statistikk.Stønad.Vedtaksresultat.INNVILGET
-                VedtakType.ENDRING -> Statistikk.Stønad.Vedtaksresultat.INNVILGET
-                VedtakType.OPPHØR -> Statistikk.Stønad.Vedtaksresultat.OPPHØRT
-                else -> throw RuntimeException("Ugyldig vedtaksresultat")
+            vedtakstype = vedtakstype(vedtak),
+            vedtaksresultat = when (vedtak) {
+                is Vedtak.EndringIYtelse.GjenopptakAvYtelse -> TODO("Ikke implementert for ${vedtak::class}")
+                is Vedtak.EndringIYtelse.InnvilgetRevurdering -> Statistikk.Stønad.Vedtaksresultat.INNVILGET
+                is Vedtak.EndringIYtelse.InnvilgetSøknadsbehandling -> Statistikk.Stønad.Vedtaksresultat.INNVILGET
+                is Vedtak.EndringIYtelse.OpphørtRevurdering -> Statistikk.Stønad.Vedtaksresultat.OPPHØRT
+                is Vedtak.EndringIYtelse.StansAvYtelse -> TODO("Ikke implementert for ${vedtak::class}")
             },
             behandlendeEnhetKode = "4815",
             ytelseVirkningstidspunkt = ytelseVirkningstidspunkt,
             gjeldendeStonadVirkningstidspunkt = vedtak.behandling.periode.fraOgMed,
             gjeldendeStonadStopptidspunkt = vedtak.behandling.periode.tilOgMed,
             gjeldendeStonadUtbetalingsstart = vedtak.behandling.periode.fraOgMed,
             gjeldendeStonadUtbetalingsstopp = vedtak.behandling.periode.tilOgMed,
-            månedsbeløp = vedtak.beregning.getMånedsberegninger().map {
-                Statistikk.Stønad.Månedsbeløp(
-                    måned = it.periode.fraOgMed.toString(),
-                    stonadsklassifisering = stønadsklassifisering(vedtak.behandling, it),
-                    bruttosats = it.getSatsbeløp().toLong(),
-                    nettosats = it.getSumYtelse().toLong(),
-                    inntekter = it.getFradrag().map { fradrag ->
-                        Statistikk.Inntekt(
-                            inntektstype = fradrag.fradragstype.toString(),
-                            beløp = fradrag.månedsbeløp.toLong(),
-                        )
-                    },
-                    fradragSum = it.getSumFradrag().toLong(),
-                )
+            månedsbeløp = when (vedtak) {
+                is Vedtak.EndringIYtelse.GjenopptakAvYtelse -> TODO("Ikke implementert for ${vedtak::class}")
```

Comment:
Dersom vi ikke skal sende statistikk for disse bør vi kanskje heller ha overloads av de vi støtter?